features/mbedtls/targets/TARGET_CRYPTOCELL310/aes_alt.c

```diff
@@ -938,7 +964,7 @@ int mbedtls_aes_crypt_ecb( mbedtls_aes_context *ctx,
 /*
  * AES-CBC buffer encryption/decryption
  */
-static int mbedtls_aes_crypt_sw_cbc( struct sw_aes_context *ctx,
+static int mbedtls_aes_crypt_sw_cbc( mbedtls_aes_context *ctx,
```

Comment:
I am not sure it will be more readable, for example, the public API is `mbedtls_aes_crypt_cbc`, and if the cipher is done in sw mode, then it will directly call `mbedtls_aes_crypt_sw_cbc` . I think in the current naming, it is more readable and understandable what is sw and what is accelerated